spec/features/admin/protocol_landing_page_spec.rb

```diff
@@ -29,8 +28,7 @@
         protocol_form.update_protocol_followup_days(40)
         protocol_detail.verify_updated_followup_days("40")
         visit admin_protocols_path
-        find(:xpath, "//td/a[text()='#{var_protocol.name}']/../../td[2]").text.include?("40")
+        find(:xpath,"//a[text()='#{var_protocol.name}']/../../..//a[contains(@class,'btn-outline-primary')]").text.include?("40")
```

Comment:
Does this need to use xpath?

Reply:
look at https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession:within